taf/api/utils/_roles.py

```diff
@@ -18,6 +17,11 @@
 from taf.models.types import Role
 from taf.log import taf_logger
 
+try:
+    from taf.yubikey import get_key_serial_by_id
+except ImportError:
+    pass
+
```

Comment:
question: Won't we need taf yubikey to work with setting up roles?

Reply:
We won't need them if we only want to use keystore files. We are not able to run any commands without yubikey manager unless this import is moved directly to the function that is using it or surrounded with try/except. If you do not have yubikey manager installed, a warning message saying that you are not able to use yubikeys will be shown when you run any of taf commands: "WARNING: yubikey-manager dependency not installed. You will not be able to use YubiKeys." I have not tested what it the error will actually look like if you try setting up roles with no yubikey manager